account_vat_period_end_statement/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,40 @@
+**Italiano**
+
+    Per generare i periodi della dichiarazione IVA,
+    aprire Impostazioni > Fatturazione > Configurazione > Intervalli data > Generazione intervalli Data e seleziona:
+
+    * Prefisso nome intervallo: Prefisso identificativo per i periodi da generare (tipicamente l'anno)
```

Comment:
Sia qui che in alcuni punti elenco successivi, dopo : andrebbe usata la lettera minuscola.
s/Prefisso/prefisso

Reply:
> Sia qui che in alcuni punti elenco successivi, dopo : andrebbe usata la lettera minuscola.
> s/Prefisso/prefisso

Questa penso ti sia sfuggita.

Reply:
Hai ragione @primes2h mi era sfuggita: mi ha distratto la P maiuscola di inizio frase